apps/condo/domains/user/schema/ResetUserService.js

```diff
@@ -98,7 +98,7 @@ const ResetUserService = new GQLCustomSchema('ResetUserService', {
                     phone: null,
                     email: null,
                     password: null,
-                    name: DELETED_USER_NAME,
+                    name: deleteName && DELETED_USER_NAME,
```

Comment:
will it work if you pass `false` in `deleteName`?

Reply:
update logic, now it works correctly

Reply:
Maybe add a test for this case?